### PR TITLE
Updated azure-cosmos SDKs and connectors to get jackson dependency from azure-core

### PR DESCRIPTION
**Pull request by kushagraThapar**
Updated azure-cosmos SDKs and connectors to get jackson dependency from azure-core

### CODE REVIEW

File: sdk/cosmos/azure-spring-data-cosmos/pom.xml [hunk 1]
```diff
@@ -100,11 +100,6 @@
       <artifactId>jackson-datatype-jdk8</artifactId>
       <version>2.13.4</version> <!-- {x-version-update;com.fasterxml.jackson.datatype:jackson-datatype-jdk8;external_dependency} -->
     </dependency>
-    <dependency>
```
Comment (xinlian12) [inline sdk/cosmos/azure-spring-data-cosmos/pom.xml:103]: what about the two jackson dependencies above?` jackson-module-parameter-names` and `jackson-datatype-jdk8`

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

File: sdk/cosmos/azure-spring-data-cosmos/pom.xml (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-spring-data-cosmos/pom.xml:103]: Those are not part of azure-core or any of the libraries that spring-data-cosmos inherits. That's why had to leave them there as it is. Just removed the commons ones (coming from azure-core).

**Review by alzimmermsft — APPROVED** [review]

**Pull request merged**